src/runtime_src/core/common/config_reader.h

```diff
@@ -303,6 +305,13 @@ get_opencl_device_counter()
   return value;
 }
 
+inline bool
+get_device_counter()
+{
+  static bool value = detail::get_bool_value("Debug.device_counter", false);
```

Comment:
@jvillarre @pgschuey would "device_counters" be more accurate then "device_counter"?